projects/uitdatabank/docs/entry-api/project-flow.md

```diff
@@ -0,0 +1,60 @@
+---
+stoplight-id: 73ef6e15ee3ec
+---
+
+# Project flow from test to production
+
+For the integration with the UiTdatabank Entry API, there is a flow in which you first obtain access to our test environment. 
+Access to production environment is given after your integration on test environment has been successful validated by publiq vzw.
+
+On this page we describe how this flow works and how you can ensure that you gain access to the production environment as quickly as possible!
+
+> Make sure you read this guide before you start developing on our test environment!
+
+## Step 1: start on test environment
+
+You can apply for free test credentials at [Projectaanvraag](https://projectaanvraag.uitdatabank.be). 
+
+Once you've obtained your personal test credentials you can start with the development of your integration with the UiTdatabank Entry API on our test environment.
+
+## Step 2: validation by publiq vzw
+
+As soon as you have finished developing your integration with the Entry API, you send a minimum of 5 of events to our test environment. You contact us via `vragen@publiq.be` and provide us the identifiers of the created test events. We will then validate your integration as soon as possible. 
+
+In this validation process we assess the integration in two manners:
+1. Is the **created data** qualitative, relevant and valuable for public agendas such as www.uitinvlaanderen.be?
+2. Is the **integration made robust**, e.g. is it possible to update and delete content through the integration?
+
+It is important that you go through the checklist below in advance and validate your integration against it. By doing this in advance, you ensure a smooth transition to the production environment!
+
+### Checklist for a succesfull integration
+#### Mandatory fields
+- [ ] **name**: only use plain text 
+- [ ] **eventtype**: use a valid ID from the [UiTdatabank taxonomy](taxonomy.uitdatabank.be/terms)
```

Comment:
Also, this should always be enforced on a technical level? We have validation for that in the backend. If that's broken we should fix it.

Reply:
I saw your comment about that in Confluence. This "tip" is based on the experience that integrators somehow expect that they can use whatever event type (both label and identifier), see the example I gave in Confluence too.
Maybe integrators just copy / use the internal logic (categorisation) of their application when they export events to UiTdatabank? Or the concept of a "taxonomy" is not clear for integrators.

Reply:
I understand. The documentation about the `terms` property now contains:

> Terms are pre-defined and can be browsed in our [Terms](https://docs.publiq.be/docs/docs/terms.md) guide.

We of course need to update that terms guide to no longer refer to the old documentation. And we can also document in future guides like "how to create an event" that the term ids are predefined. So I think if we mention it in all those places, it should be clear enough that we don't need to mention this constraint that is already enforced automatically here in a list of "soft" constraints?

Because an integrator will always get back a 400 when they use an uknown term id, and when that happens and they go looking for a solution I assume they're more likely to look at e.g. the terms guide, or "how to create an event" than the content checklist.

Of course it doesn't really harm here, but I think it's useful to keep this checklist limited to "soft" constraints that we don't enforce on a technical level and have "debugging" info for technical constraints in other places where integrators are more likely to look?